apps/protocol-api/src/main.ts

```diff
@@ -153,6 +155,13 @@ const permissions = shield(
       name: or(isAuthenticated, hasToken),
       users: or(isAuthenticated, hasToken),
     },
+    Chain: {
+      id: hasToken,
```

Comment:
why no isAuthenticated?

Reply:
Because it is not used by the frontend, best to only add permissions when needed.